### PR TITLE
12974 display owner status in selection list and results

### PR DESCRIPTION
**Pull request by ChasBoucher**
*Issue #:* /bcgov/entity12974

Change status to Owner Status in MHR Search results

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/tables/mhr/SearchedResults.vue [hunk 1]
```diff
@@ -124,7 +124,7 @@
             {{ item.mhrNumber }}
           </template>
           <template v-slot:[`item.state`]="{ item }">
-            {{ item.status }}
+            {{ getOwnerStatus(item.ownerStatus) }}
```
Comment (cameron-eyds) [inline ppr-ui/src/components/tables/mhr/SearchedResults.vue:127]: Was `status` too  `ownerStatus` a recent property name update done on the BE?

File: ppr-ui/src/components/tables/mhr/SearchedResults.vue [hunk 2]
```diff
@@ -290,6 +290,19 @@ export default defineComponent({
       } else return '-'
     }
 
+    const getOwnerStatus = (ownerStatus: string): string => {
+      if (ownerStatus) {
+       if(ownerStatus=='PREVIOUS')
```
Comment (cameron-eyds) [inline ppr-ui/src/components/tables/mhr/SearchedResults.vue]: Minor spacing issues (linter might complain) 
also the equality operator we use traditionally  is the strict equality (`===`), which also checks for type. 
The `==` equality operator will type coerce properties. 
Super small thing but good for consistency 👍 

`if (ownerStatus === 'PREVIOUS')`

File: ppr-ui/src/components/tables/mhr/SearchedResults.vue [hunk 3]
```diff
@@ -290,6 +290,19 @@ export default defineComponent({
...
+       {
+         return 'HISTORICAL'
+       } 
+       else
+         return ownerStatus
+      }
+      else
+         return ''
```
Comment (cameron-eyds) [inline ppr-ui/src/components/tables/mhr/SearchedResults.vue]: Minor syntax clean up, although not wrong, i would keep the `else` on the same line as the closing bracket. 
Also add brackets for the statement or keep the return on the same line. 

ie `} else return ownerStatus` or `} else { return ownerStatus }`
Some people prefer the brackets for consistency, i'm good with either.

**Review by cameron-eyds — COMMENTED** [review]
Nice job!
Just a few comments/questions
Also see the lint issues in the ci.

**Review by doug-lovett — APPROVED** [review]
No comments in addition to Cameron's.

File: ppr-ui/src/components/tables/mhr/SearchedResults.vue (hunk 2 above)
Comment (ChasBoucher) [reply ppr-ui/src/components/tables/mhr/SearchedResults.vue]: Good point.

File: ppr-ui/src/components/tables/mhr/SearchedResults.vue (hunk 1 above)
Comment (ChasBoucher) [reply ppr-ui/src/components/tables/mhr/SearchedResults.vue:127]: @cameron-freshworks I am not sure what BE is, but ownerStatus was the field that should have been used.  The api was returning it.  It is the owner.status.  Ali confirmed it.

Comment (cameron-eyds) [reply ppr-ui/src/components/tables/mhr/SearchedResults.vue:127]: Sounds good, thanks!

**Review by cameron-eyds — APPROVED** [review]
Thank you! Looks good 👍

**Pull request merged**